_data/lang/en.yml

```diff
@@ -197,8 +197,16 @@ constants:
 
 # for contact form
 contact_form:
-  button_name: Contact Form
-  formLoadingText: Loading the Contact Form...
+  button_name: Submit
+  name: Name
+  email: Email
+  message: Message
+  alert: The form has sent correctly
```

Comment:
It should be `The form has been sent correctly`. But better ask the client what messages do they want in the different languages.